sdcm/sct_events/loaders.py

```diff
@@ -239,13 +239,23 @@ def msgfmt(self):
 
 class ScyllaBenchLogEvent(LogEvent, abstract=True):
     ConsistencyError: Type[LogEventProtocol]
+    DataValidationError: Type[LogEventProtocol]
+    ParseDistributionError: Type[LogEventProtocol]
 
 
 ScyllaBenchLogEvent.add_subevent_type("ConsistencyError", severity=Severity.ERROR, regex=r"received only")
+# Scylla-bench data validation was added by https://github.com/scylladb/scylla-bench/commit/3eb53d8ce11e5ad26062bcc662edb31dda521ccf
+ScyllaBenchLogEvent.add_subevent_type("DataValidationError", severity=Severity.CRITICAL,
+                                      regex=r"doesn't match |failed to validate data|failed to verify checksum|corrupt checksum or data|"
+                                            r"data corruption")
+ScyllaBenchLogEvent.add_subevent_type("ParseDistributionError", severity=Severity.CRITICAL,
+                                      regex=r"invalid distribution specifification")
```

Comment:
Is the typo `specifification` also in the SB output?

Reply:
It is typo in the commit: https://github.com/scylladb/scylla-bench/commit/3eb53d8ce11e5ad26062bcc662edb31dda521ccf#diff-e93c66355653cdd1f874d83fda442afc98cbafacc3d558477f36cfd10ec32eed

Reply:
Then please update the regex to support both variants - typo'ed and proper one for case it gets fixed in the scylla-bench.

Reply:
Changed regex according to current error messages (https://github.com/scylladb/scylla-bench/blob/12020ff90cacd0dbfabebfe488029978429f3efa/random/random.go#L140)